### ISSUE TITLE
Implement Alexa skill

### ISSUE THREAD

**Issue by juliaqiuxy** [juliaqiuxy/slopeninja-frontend#17]
Deploy the Alexa skill as soon as alexa/alexa-skills-kit-sdk-for-nodejs#533 is resolved.

**Comment by alexindigo** [comment]
How about facebook messenger bot? :)

**Comment by juliaqiuxy** [comment]
@alexindigo Great idea! I should probably take a look into that next. 🙋🏻

**Comment by juliaqiuxy** [comment]
@alexindigo Great idea! I should probably look into that next. 🙋🏻

**Comment by juliaqiuxy** [comment]
Just saw [alexindigo/fbbot](https://github.com/alexindigo/fbbot). Are you still maintaining it?

**Comment by alexindigo** [comment]
Yeah, it's been somewhat rusty since T. abandoned it's messenger bot. :) Shouldn't be hard to update to add latest protocol features. :)
